src/transcripts/tabnine.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Ron X & Yotam Shinan
```

Comment:
These def still need more details added before we can merge

Reply:
Thursday Jan 27 @ Noon PT
Tabnine is building a top-to-bottom AI-assisted development workflow that empowers all code creators, in all languages, from concept through to completion.

I would just say Tabnine Team and use the https://twitter.com/Tabnine_ as their handle.